bin/launch-process

```diff
@@ -0,0 +1,282 @@
+#!/usr/bin/env bash
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+set -e
+
+BIN=$(cd "$( dirname "$( readlink "$0" || echo "$0" )" )"; pwd)
+
+USAGE="Usage: launch-process <process name> [-c]
+
+launch-process launches one of the Alluxio processes in the foreground. By
+default it will only log to the configured log file. To configure the process,
+add configuration properties in alluxio-site.properties or environment
+variables in conf/alluxio-env.sh.
+
+Note that this script does not verify Alluxio configuration before starting.
+It simply populates the necessary environment variables and launches the
+desired process in the foreground. It is recommended to use alluxio-start.sh
+unless you know what you are doing.
+
+    <process name>    One of: master, secondary_master, job_master, worker,
+                      job_worker, proxy, logserver
+
+    -c                Provide this flag to also log directly to stdout
+
+    -h                Print this help text to stdout
+"
+
+# Launches an Alluxio process
+#
+# Arguments:
+#         1: any java opts to pass to the process
+#         2: the main class to launch
+#         3.... Additional arguments to the java program
+launch_process() {
+  local java_opts
+  local main_class
+
+  java_opts=${1}
+  main_class=${2}
+
+  exec ${JAVA} -cp ${ALLUXIO_SERVER_CLASSPATH} ${java_opts} "${main_class}" ${@:3}
+}
+
+# Launch a master process
+#
+# Environment Variables:
+#     JOURNAL_BACKUP: (Optional) A location of a journal backup. If set, the
+#                     master will read the value of this variable as a journal
+#                     backup.
+launch_master() {
+  if [[ -n ${JOURNAL_BACKUP} ]]; then
+    ALLUXIO_MASTER_JAVA_OPTS+=" -Dalluxio.master.journal.init.from.backup=${JOURNAL_BACKUP}"
+  fi
+
+  # use a default Xmx value for the master
+  contains "${ALLUXIO_MASTER_JAVA_OPTS}" "Xmx"
+  if [[ $? -eq 0 ]]; then
+    ALLUXIO_MASTER_JAVA_OPTS+=" -Xmx8g "
+  fi
```

Comment:
There is a new option introduced in #9697 , could you add that as well?

Reply:
done